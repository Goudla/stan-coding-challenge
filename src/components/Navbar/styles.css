.navbar {
  background-image: linear-gradient(to bottom, #09f 0, #06f 100%);
  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.5);
  height: 60px;
  position: fixed;
  top: 0;
  width: 100%;
  z-index: 10;
}
