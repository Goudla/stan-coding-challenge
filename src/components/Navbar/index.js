import * as React from 'react';

// Styles
import './styles.css';

type Props = {
  children: React.Node
};

class Navbar extends React.PureComponent<Props> {
  render() {
    const { children } = this.props;

    return (
      <div className="navbar">
        {children}
      </div>
    );
  }
}

export default Navbar;
