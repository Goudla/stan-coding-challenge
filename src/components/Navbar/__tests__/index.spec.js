import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import Navbar from '../';

describe('<Navbar />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Navbar>Test</Navbar>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
