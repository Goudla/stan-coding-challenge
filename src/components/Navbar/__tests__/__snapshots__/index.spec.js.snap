// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> Should render 1`] = `
<div
  className="navbar"
>
  Test
</div>
`;
